.github/CONTRIBUTING.md

```diff
@@ -43,35 +45,37 @@ Botpress is an open-source project and we welcome any contributions from the com
 
 Those are all great contributions and will be welcomed as much as any other.
 
-# Your First Contribution in Botpress
-If you're not sure where to start issues we've labeled some issues with [#for-new-contributors](https://github.com/botpress/botpress/issues?q=is%3Aissue+is%3Aopen+label%3Afor-new-contributors) so these are a great place to start!
```

Comment:
You have no label like this, so I assume it's a mistake and you meant "good first issue".